test/e2e/dra/README.md

```diff
@@ -36,6 +36,180 @@ $ kind build node-image --image dra/node:latest $(pwd)
+
+
+$ kind build node-image --image kindest/node:${version//+/_} $(pwd)
+
+# CAPI needs access to Docker.
+$ cat >kind-cluster-with-extramounts.yaml <<EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: dual
+nodes:
+- role: control-plane
+  extraMounts:
+    - hostPath: /var/run/docker.sock
+      containerPath: /var/run/docker.sock
+EOF
+
+$ kind create cluster --config kind-cluster-with-extramounts.yaml
+...
+Creating cluster "kind" ...
+...
+
+$ CLUSTER_TOPOLOGY=true clusterctl init --infrastructure docker
+Fetching providers
+Installing cert-manager Version="v1.13.2"
+Waiting for cert-manager to be available...
+Installing Provider="cluster-api" Version="v1.6.0" TargetNamespace="capi-system"
+Installing Provider="bootstrap-kubeadm" Version="v1.6.0" TargetNamespace="capi-kubeadm-bootstrap-system"
+Installing Provider="control-plane-kubeadm" Version="v1.6.0" TargetNamespace="capi-kubeadm-control-plane-system"
+Installing Provider="infrastructure-docker" Version="v1.6.0" TargetNamespace="capd-system"
+
+Your management cluster has been initialized successfully!
+
+You can now create your first workload cluster by running the following:
+
+  clusterctl generate cluster [name] --kubernetes-version [version] | kubectl apply -f -
+
+# Patching in the configuration for pod security conflicts with
+# extraArgs for apisever due to this JSON patch:
+#     - jsonPatches:
+#      - op: add
+#        path: /spec/template/spec/kubeadmConfigSpec/clusterConfiguration/apiServer/extraArgs
+#        value:
+#         admission-control-config-file: /etc/kubernetes/kube-apiserver-admission-pss.yaml
+
+$ POD_SECURITY_STANDARD_ENABLED=false clusterctl generate cluster capi-quickstart --flavor development   --kubernetes-version $version --control-plane-machine-count=1   --worker-machine-count=1 >cluster-latest.yaml
+
+# MachinePools are experimental, so this patch removes them from the cluster.
+# Autoscaling will be done through MachineDeployments.
+
+$ cat cluster.patch <<EOF
+*** cluster-1.30.0.yaml	2024-01-15 18:03:14.507591268 +0100
+--- cluster-1.30.0-dra-II.yaml	2024-01-15 19:28:02.918296629 +0100
+***************
+*** 250,255 ****
+--- 250,258 ----
+        kubeadmConfigSpec:
+          clusterConfiguration:
+            apiServer:
++             extraArgs:
++               runtime-config: resource.k8s.io/v1alpha2=true
++               feature-gates: DynamicResourceAllocation=true,ContextualLogging=true
+              certSANs:
+              - localhost
+              - 127.0.0.1
+***************
+*** 258,263 ****
+--- 261,271 ----
+            controllerManager:
+              extraArgs:
+                enable-hostpath-provisioner: "true"
++               feature-gates: DynamicResourceAllocation=true,ContextualLogging=true
++           scheduler:
++             extraArgs:
++               feature-gates: DynamicResourceAllocation=true,ContextualLogging=true
++         # TODO: enable features in kubelet
```

Comment:
Should be solved by just adding `kubeletExtraArgs` to the `initConfiguration` (and joinConfiguration?) below?